src/main/java/frc/robot/joystickManager/JoystickState.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.joystickManager;
+
+public enum JoystickState {
```

Comment:
im not sure its so smart it will be an enum... for the second joystick, there shouldn't be an option for swerve since it's not controloing it. consider changing it to an int (that represents the state)

Reply:
no... that's the whole point of the project. that we could make any controller control any part of the robot